app/cdap/components/shared/Table/Table.stories.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import Table from '.';
+import TableHeader from './TableHeader';
+import TableRow from './TableRow';
+import TableCell from './TableCell';
+import TableBody from './TableBody';
+
+export default {
+  title: 'Listing Table',
```

Comment:
This is called `Listing Table` because it's meant for lists of entities with detail columns. Other possible table components would be for tabular data, file browsing (see https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/components/Connections/Browser/GenericBrowser/BrowserTable.tsx), etc.